Dockerfile.armhf

```diff
@@ -0,0 +1,20 @@
+FROM resin/armhf-alpine:3.5
```

Comment:
Interesting thing about resin/armhf-alpine is that its based on `armhf/alpine` with quemu support transparent.